2022-01-31 16:18:09,609 INFO    Thread-3  :222084 [wandb_setup.py:_flush():71] setting env: {'project': 'pytorch-sweeps-demo', 'entity': 'paulina', 'root_dir': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10', 'sweep_id': 'o8viijca', 'run_id': 'ti6l2u7l', 'sweep_param_path': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/sweep-o8viijca/config-ti6l2u7l.yaml'}
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_init.py:_log_setup():371] Logging user logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_161809-ti6l2u7l/logs/debug.log
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_init.py:_log_setup():372] Logging internal logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_161809-ti6l2u7l/logs/debug-internal.log
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_init.py:init():404] calling init triggers
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 55, 'epochs': 1, 'layer1_size': 128, 'layer2_size': 512, 'learning_rate': 0.06232720497790063, 'optimizer': 'sgd'}
config: {}
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [wandb_init.py:init():460] starting backend
2022-01-31 16:18:09,610 INFO    Thread-3  :222084 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-31 16:18:09,620 INFO    Thread-3  :222084 [backend.py:ensure_launched():216] starting backend process...
2022-01-31 16:18:09,629 INFO    Thread-3  :222084 [backend.py:ensure_launched():221] started backend process with pid: 222215
2022-01-31 16:18:09,640 INFO    Thread-3  :222084 [wandb_init.py:init():469] backend started and connected
2022-01-31 16:18:09,641 INFO    Thread-3  :222084 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 55, 'epochs': 1, 'layer1_size': 128, 'layer2_size': 512, 'learning_rate': 0.06232720497790063, 'optimizer': 'sgd'}
2022-01-31 16:18:09,642 INFO    Thread-3  :222084 [wandb_init.py:init():533] updated telemetry
2022-01-31 16:18:09,648 INFO    Thread-3  :222084 [wandb_init.py:init():563] communicating current version
2022-01-31 16:18:10,648 INFO    Thread-3  :222084 [wandb_init.py:init():568] got version response 
2022-01-31 16:18:10,648 INFO    Thread-3  :222084 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-31 16:18:10,816 INFO    Thread-3  :222084 [wandb_init.py:init():606] starting run threads in backend
2022-01-31 16:18:12,817 INFO    Thread-3  :222084 [wandb_run.py:_console_start():1810] atexit reg
2022-01-31 16:18:12,817 INFO    Thread-3  :222084 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-31 16:18:12,818 INFO    Thread-3  :222084 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-31 16:18:12,818 ERROR   Thread-3  :222084 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-01-31 16:18:12,820 INFO    Thread-3  :222084 [wandb_init.py:init():633] run started, returning control to user process
2022-01-31 16:19:11,323 INFO    Thread-3  :222084 [wandb_run.py:finish():1457] finishing run paulina/pytorch-sweeps-demo/ti6l2u7l
2022-01-31 16:19:11,324 INFO    Thread-3  :222084 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-31 16:19:11,325 INFO    Thread-3  :222084 [wandb_run.py:_restore():1752] restore
2022-01-31 16:19:11,331 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 604
  total_bytes: 604
}

2022-01-31 16:19:11,795 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 604
  total_bytes: 604
}

2022-01-31 16:19:11,942 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 604
  total_bytes: 604
}

2022-01-31 16:19:12,044 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 604
  total_bytes: 7761
}

2022-01-31 16:19:12,145 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 604
  total_bytes: 7761
}

2022-01-31 16:19:12,246 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2122
  total_bytes: 7761
}

2022-01-31 16:19:12,347 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}

2022-01-31 16:19:12,448 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}

2022-01-31 16:19:12,548 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}

2022-01-31 16:19:12,649 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}

2022-01-31 16:19:12,981 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}

2022-01-31 16:19:13,311 INFO    Thread-3  :222084 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7761
  total_bytes: 7761
}
local_info {
}

2022-01-31 16:19:14,410 INFO    Thread-3  :222084 [wandb_run.py:_append_files():2180] logging synced files
